### PR TITLE
:sparkles: Add Monte-Carlo Batch Normalization

### PR DESCRIPTION
**Pull request by o-laurent**
Also rework Scalers, Imagenet and ImageNet-O

- [x] Move the wrapper to post-processing
- [x] Simplify the code by externalizing the logic
- [x] Add documentation
- [x] Add ref
- [x] Add tests
- [x] Add a tutorial
- [x] Improve coverage

### CODE REVIEW

File: tests/_dummies/model.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 ]
 
 
+class Identity(nn.Module):
```
Comment (alafage) [inline tests/_dummies/model.py:9]: It already exists in PyTorch: `torch.nn.Identity()`.

**Review by alafage — APPROVED** [review]
It looks great @o-laurent, nicely done! Maybe use `nn.Identity` for the test scripts instead of redefining the class.

**Comment by o-laurent** [comment]
> It looks great @o-laurent, nicely done! Maybe use `nn.Identity` for the test scripts instead of redefining the class.

Done! Thanks, @alafage, for your sharp eyes!
